### PR TITLE
Testing Calculator with Jest

### PR DESCRIPTION
**Pull request by MkrtichSargsyan**
In this milestone, I have added tests that check all functions in calculate.js and added 
snapshots which are checking if the main components are displayed

### CODE REVIEW

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 
@MkrtichSargsyan  well done on this milestone, Almost all the project requirement is met, however, few changes are required
#### highlight

- all functionalities are tested using React Testing Library, @testing-library/react, see the screenshot from Pathwright. 👍🏻 


### improve

- [ ] all functions in `calculate.js` file are covered with unit tests e.g `+`,`+/-`,`-` and `/` are  yet to be tested kindly test them 

# once specific changes are made kindly request another review

File: src/__tests__/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+// import { render, screen, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import calculate from '../logic/calculate';
+
```
Comment (eliasecasta) [inline src/__tests__/calculate.test.js:4]: - [ ] Please also add test cases for "+/-" and "/" as requested by the previous CR, also, don't forget a test for division of a number by 0 and negative tests.

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hi @MkrtichSargsyan, 

Great work on making the changes requested by a previous reviewer like the "+" and "-" test cases 👏 

You've done well implementing some of the requested changes, but some are still not addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/__tests__/calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
+// import { render, screen, cleanup } from '@testing-library/react';
```
Comment (bellom) [inline src/__tests__/calculate.test.js:1]: let's remove commented code. 
It’s better to delete commented code. If you need to review a previous version of your code, use git history.

I trust you can remove this before merging.

Sorry for some reason it wasn't mentioned in the previous review.

**Review by bellom — APPROVED** [review]
## Hi @MkrtichSargsyan  👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
